src/main/validator.ts

```diff
@@ -48,13 +57,17 @@ const validatorLogs = async (msg: ValidatorLogsRequest) => {
   const MAX_DISPLAY_LINES = 30;
 
   if (net !== Net.Localhost) {
-    return '';
+    return `Cannot show validator container output from ${net}`;
```

Comment:
IIRC this should be a proper `throw new Error(`Cannot show validator container output from ${net}`)` and lint might well complain about it